content/en/tests/flaky_management/_index.md

```diff
@@ -116,18 +116,26 @@ Flaky Test Management uses AI to automatically assign a root cause category to e
 | **Environment Dependency** | Test depends on specific OS, library versions, or hardware. It may pass on one environment but fail on another, especially in cloud-CI environments where machines vary nondeterministically. |
 | **Unknown**             | Test is flaky for an unknown reason. |
 
+## Receive notifications
+
+Set up notifications to avoid missing changes to your flaky tests. Whenever a user or a policy changes the state of a flaky test, a message is sent to your selected recipients. You can send notifications to email addresses or Slack channels (see the [Datadog Slack integration][5]) and route messages based on test code owners. If no code owners are specified, all selected recipients are notified of all flaky test changes in the repository. Notifications are configured per repository from the  [**Flaky Test Policies**][12] page in Software Delivery settings.
+
+{{< img src="tests/flaky_management_notifications_settings.png" alt="Notifications settings UI" style="width:100%;" >}}
+
+Notifications are not sent immediately; they are batched every few minutes to reduce noise.
```

Comment:
Recommend moving this up to above the image so it's not lost. See the above suggestion:
`<div class="alert alert-info">Notifications are not sent immediately but are instead batched every few minutes to reduce noise.</div>`

```suggestion
```